src/Form/Type/Api/AddPackageType.php

```diff
@@ -57,6 +58,9 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ->add('keepLastReleases', IntegerType::class, [
                 'data' => 0,
                 'required' => false,
+                'constraints' => [
+                    new PositiveOrZero(),
```

Comment:
should we also make sure that someone does not enter a decimal value? ... :thinking: although it probably does not matter because they will round off anyway when compared

Reply:
Very good :+1: